packages/office-ui-fabric-react/src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -161,13 +162,16 @@ export class ChoiceGroup extends React.Component<IChoiceGroupProps, IChoiceGroup
   }
 
   private _onChange(option: IChoiceGroupOption, evt: React.FormEvent<HTMLInputElement>) {
-    let { onChanged } = this.props;
+    let { onChanged, onChange } = this.props;
 
     this.setState({
       keyChecked: option.key
     });
 
-    if (onChanged) {
+    // TODO: onChanged deprecated, remove else if after 07/17/2017 when onChanged has been removed.
+    if (onChange) {
+      onChange(evt, option);
```

Comment:
Not sure the reason behind param order flip but most of the time we are more interested in changed `option` instead of `event` so its better to keep it first.